airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2718,10 +2718,6 @@ components:
             $ref: "#/components/schemas/OperationId"
         syncCatalog:
           $ref: "#/components/schemas/AirbyteCatalog"
-        schedule:
```

Comment:
The schedule should still exist even if the nullable is moved.

Reply:
Oh oops, accidentally removed it entirely. Will update